stl/inc/iterator

```diff
@@ -617,10 +611,7 @@ public:
         _CATCH_END
     }
 
-    // clang-format off
-    constexpr counted_iterator operator++(int)
-        requires forward_iterator<_Iter>
-    {
+    constexpr counted_iterator operator++(int) requires forward_iterator<_Iter> {

```

Comment:
I was wondering about this during the iterator PRs a lot. Loving it! (Now that I really thin about it I believe there is one occurrence with spans default constructor that already does this.)

Reply:
If you're referring to the *trailing-requires-clause* that's on the same line as the rest of the function declaration, there are a bunch of examples in `subrange` as well.